pl_bolts/datasets/dummy_dataset.py

```diff
@@ -94,14 +105,25 @@ class RandomDictDataset(Dataset):
         >>> from torch.utils.data import DataLoader
         >>> ds = RandomDictDataset(10)
         >>> dl = DataLoader(ds, batch_size=7)
+        >>> batch = next(iter(dl))
+        >>> len(batch['a']),len(batch['a'][0])
+        (7, 10)
+        >>> len(batch['b']),len(batch['b'][0])
+        (7, 10)
     """
 
     def __init__(self, size: int, num_samples: int = 250):
         """
         Args:
-            size: tuple
+            size: integer representing the length of a feature_vector
             num_samples: number of samples
         """
+        if num_samples < 1:
+            raise ValueError("Provide an argument greater than 0 for `num_samples`")
+
+        if size < 1:
+            raise ValueError("Provide an argument greater than 0 for `size`")
+
         self.len = num_samples
         self.data = torch.randn(num_samples, size)
```

Comment:
can `b` in the dictionary also be random and not dependant on `a`?

Reply:
made the commit with the necessary changes.thanks!